components/kubeflow/kfserving/src/kfservingdeployer.py

```diff
@@ -414,6 +414,14 @@ def main():
             # Remove some less needed fields to reduce output size.
             del model_status['metadata']['managedFields']
             del model_status['status']['conditions']
+            if sys.getsizeof(model_status) > 3000:
+                del model_status['address']['url']
```

Comment:
```suggestion
```
@nachogarciam everything looks good. However, often time I see some of our pipelines are relying on this address url to do some prediction in subsequent tasks. Since removing the address.url won't save us that much space, maybe we can remove this line?

Reply:
Thanks Tommy, done!